test/engine_tests.jl

```diff
@@ -0,0 +1,73 @@
+include("old_engine/NurseScheduling.jl")
+include("../src/repair_schedule.jl")
+
+using Test
+using .NurseSchedules:
+    Schedule,
+    get_disallowed_sequences
+
+function repair!(schedule)
+    fix = repair_schedule(schedule.data)
+    update_shifts!(schedule, fix)
+end
+
+function solve(file)
+    schedule = Schedule(file)
+    redirect_stdout(open("/dev/null", "w")) do
+        repair!(schedule)
+    end
+    schedule.data
+end
+
+function check_errors(file)
+    new_errors = get_errors("schedules/schedule_2016_august_medium.json")
+
+    old_errors = OldNurseSchedules.get_errors("schedules/schedule_2016_august_medium.json")
+    test_flag = true
+    for old_error in old_errors
+        new_error = new_errors[1]
+        if old_error != "AON"
+            if old_error == new_error
+                deleteat!(new_errors, 1)
+            else
+                test_flag = false
+            end
+        else
+            if old_error["day"] == new_error["day"]
+                deleteat!(new_errors, 1)
+            else
+                test_flag = false
+            end
+        end
+    end
+    test_flag
```

Comment:
Czy trzeba fory robić, nie wystarczy `new_error == old_errors`? A jak nie to lepiej zrobic jakiś preprocessing, zeby tak można było.

Reply:
Kwestia zmienionego sposobu raportowania AON, to mogę podmienić na preprocessing.